codecov.yml

```diff
@@ -0,0 +1,21 @@
+coverage:
+  status:
+    project:
+      default: off
+      production:
+        target: auto
```

Comment:
(thought I commented this) why use **target: auto** and not a new, call it MARCH, coverage standard for projects with code coverage?

Reply:
Same question in other PR so this one is redundant.